docs/releasing/publishing.md

```diff
@@ -1,55 +1,62 @@
+    - GOV.UK Prototype kit
+  - check who the release’s contributors are and if we have consent to include their name
 
-4. Ask a technical writer or a content designer to review the summary.
+5. Technical writer to finalise draft of release notes and get 2i on them.
 
-5. Before you proceed to the next section, confirm with the appropriate person that the release notes and any related pull requests are ready to publish. Depending on the issue, the appropriate person may be the technical writer, the content designer, or a developer. If you’re unsure who it is, then ask within the [#govuk-design-system](https://gds.slack.com/archives/CAF8JA25U) Slack channel.
+6. Content designer/Designer to update community backlog with rationale for any decisions we made.
 
-# Steps you must complete to release a new version of GOV.UK Frontend
+7. Coordinate the sign-off when you are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the []#design-system-team-channel](https://app.slack.com/client/T8GT9416G/G011V7RAAKY) to confirm sign-off from:
+  - Content and designers for guidance, examples and community backlog decision rationale
+  - Technical writer and developers for Nunjucks macros
+  - Developers for changes to GOV.UK Frontend
+  - Technical writer for release notes
+  - Content/Community Manager for announcements and engagement activities
 
-1. Checkout **master** and pull latest changes.
+> **Note:** Before you go on leave, tell the delivery manager who will be looking after your work. This will help us to complete sign-off without fuss.
 
-2. Run `nvm use` to ensure you are using the right version of Node.js and npm.
+# Release a new version of GOV.UK Frontend
 
-3. Run `npm install` to ensure you have the latest dependencies installed.
+Developers should pair on releases. When remote working, it can be useful to be on a call together.
 
-4. Create and checkout a new branch (`release-[version-number]`).
-  The version number is determined by looking at the [current "Unreleased" CHANGELOG](../../CHANGELOG.md) changes and updating the previous release number depending on the kind of entries:
+1. Checkout the **master** branch and pull the latest changes.
 
-  - `Breaking changes` corresponds to a `major` (1.X.X) change.
-  - `New features` corresponds to a `minor` (X.1.X) change.
-  - `Fixes` corresponds to a `patch` (X.X.1) change.
+2. Run `nvm use` to make sure you are using the right version of Node.js and npm.
 
-  For example if the previous version is `2.3.0` and there are entries for `Breaking changes` then the new release should be `3.0.0`.
+3. Run `npm install` to make sure you have the latest dependencies installed.
 
-  See the [versioning documentation](versioning.md) for more information.
+4. Create and checkout a new branch (`release-[version-number]`). See the [versioning documentation](versioning.md) for more information.
```

Comment:
This is not to do with the changes in this PR but I just noticed a broken link here. This should be '/docs/contributing/versioning.md' 🔍

Reply:
```suggestion
4. Create and checkout a new branch (`release-[version-number]`). See the [versioning documentation](/docs/contributing/versioning.md) for more information.
```